docs/core-fabric.md

```diff
@@ -340,6 +340,7 @@ fabric:
           finality:
             numRetries: 3
             unknownTxTimeout: 100ms
+          parallelism: 3 # maximum go routines to commit at the same time transactions of the same block
```

Comment:
seems we do 5 in the integration tests, is that a safe number? If so, maybe we could propose 5 as default?